Source/WebCore/PAL/pal/spi/mac/NSViewSPI.h

```diff
@@ -49,7 +49,9 @@ typedef NS_ENUM(NSInteger, NSViewSemanticContext) {
 @property (nonatomic, setter=_setSemanticContext:) NSViewSemanticContext _semanticContext;
 #endif
 
+#if !HAVE(NSVIEW_CLIPSTOBOUNDS_API)
 @property BOOL clipsToBounds;
```

Comment:
Any reason this property wasn't guarded before?

Reply:
clipsToBounds has been SPI for a long time, but was just promoted to API. See https://developer.apple.com/documentation/appkit/nsview/4236466-clipstobounds?language=objc&changes=latest_minor: available since 10.9, but added to the SDK in this release.